stacks/HANA-HA/playbook.yml

```diff
@@ -74,4 +74,5 @@
     vars:
       sap_hana_primary_instance_name: '{{ terraform.outputs.primary_instance_name.value }}'
       sap_hana_secondary_instance_name: '{{ terraform.outputs.secondary_instance_name.value }}'
+      sap_hana_deployment_create_initial_tenant: '{{ sap_hana_deployment_create_initial_tenant }}'
```

Comment:
This shouldn't be necessary if there is already a default and it will cause a variable lookup loop if it's not defined in the vars.

Reply:
True. Just pushed the change.